deps/llhttp/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-cmake_minimum_required(VERSION 3.25.0)
+cmake_minimum_required(VERSION 3.5.1)
 cmake_policy(SET CMP0069 NEW)
 
-project(llhttp VERSION 9.3.0)
```

Comment:
Why is the system performing a downgrade? cc @nodejs/actions @ShogunPanda

Reply:
Because of https://github.com/nodejs/node/pull/58271#issuecomment-2869611473